tensorflow/core/kernels/reduction_gpu_kernels.cu.h

```diff
@@ -151,6 +151,9 @@ struct DividesBy<std::complex<double>> {
   }
 };
 
+#endif
+
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
```

Comment:
to me it seems line 156 `#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM` and line 164 `#endif` can be removed altogether as the file itself is conditioned upon `#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM`